htdocs/comm/remx.php

```diff
@@ -1060,3 +1063,13 @@
 // End of page
 llxFooter();
 $db->close();
+?>
+<script>
+    jQuery(document).ready(function () {
+	jQuery("#discount_amount_ttc").keyup(function () {
+	var tva_tx = 1+(jQuery("#tva_tx").val()/100);
+	var value = Math.round10(jQuery(this).val()/1.18,-8);
+	jQuery("#amount_ht").val(value);
+	});
+	});
+</script>
```

Comment:
No javascript outside of the HTML content.
Please mode the javacript just before the llxFooter().

Reply:
Thank you, I will take it into account